### PR TITLE
Ne list products lots

### PR DESCRIPTION
**Pull request by ski-climb**
@tmikeschu @ethanbennett @DavidKnott Ready for review.  Needs one person.

### CODE REVIEW

File: app/controllers/api/v1/lots_controller.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
 class Api::V1::LotsController < ApplicationController
   skip_before_action :verify_authenticity_token, only: [:create] 
 
+  def index
+    product = Product.find_by_id(params[:product_id])
+    if product
+      render json: product.lot_ids
```
Comment (tmikeschu) [inline app/controllers/api/v1/lots_controller.rb:7]: Do you believe in magic?

File: spec/requests/api/v1/product_lots_spec.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    expect(response.status).to eq 200
+    expect(lots).to be_an Array
+    expect(lots.length).to eq 4
+    lots.each do |lot_id|
+      expect(product_2_lot_ids).to include(lot_id)
+      expect(product_1_lot_ids).not_to include(lot_id)
+    end
+  end
+
+  it "returns a 400 when given a nonexistent product id" do
```
Comment (tmikeschu) [inline spec/requests/api/v1/product_lots_spec.rb:29]: Good form 👍

**Review by DavidKnott — APPROVED** [review]
Great work, and way to utilize rails magic!

**Review by tmikeschu — APPROVED** [review]
Awesome!

File: app/controllers/api/v1/lots_controller.rb (hunk 1 above)
Comment (case-eee) [reply app/controllers/api/v1/lots_controller.rb:7]: i believe in magic!! 🎉

**Comment by case-eee** [comment]
good work!!!

**Pull request merged**